packages/cel/src/func.ts

```diff
@@ -194,3 +260,44 @@ export class OrderedDispatcher implements Dispatcher {
     return undefined;
   }
 }
+
+function isOfType(
+  val: unknown,
+  type: CelValueType,
+): val is TypeOf<CelValueType> {
+  if (typeof type === "object") {
+    if ("type" in type) {
+      switch (type.type) {
+        case "list":
+          return val instanceof CelList;
+        case "map":
+          return val instanceof CelMap;
+        default:
+          return false;
+      }
+    }
+    return isMessage(val, type);
+  } else {
```

Comment:
Can you remove the else? Pretty sure the linter will complain, once we enable it.